### PR TITLE
[NavigationBar] Add Accessibility method tests

### PR DESCRIPTION
**Pull request by romoore**
Just expanding the amount of unit test coverage we have.

### CODE REVIEW

**Comment by romoore** [comment]
Blocked on #2713

File: components/NavigationBar/tests/unit/NavigationBarTests.m [hunk 1]
```diff
@@ -136,4 +138,128 @@ - (void)testEncoding {
...
+  // Then
+  XCTAssertFalse(navBar.isAccessibilityElement);
+}
+
+- (void)testAccessibilityItemsCountWithNoTitle {
+  // Given
+  MDCNavigationBar *navBar = [[MDCNavigationBar alloc] init];
+
+  // Then
+  XCTAssertEqual(3, navBar.accessibilityElementCount);
```
Comment (jverkoey) [inline components/NavigationBar/tests/unit/NavigationBarTests.m]: How are we choosing this number? What would it mean if this test were to fail in the future?

Comment (jverkoey) [reply components/NavigationBar/tests/unit/NavigationBarTests.m]: May be helpful to document the number here.

**Review by jverkoey — APPROVED** [review]

File: components/NavigationBar/tests/unit/NavigationBarTests.m (hunk 1 above)
Comment (romoore) [reply components/NavigationBar/tests/unit/NavigationBarTests.m]: Canceled travis

Comment (romoore) [reply components/NavigationBar/tests/unit/NavigationBarTests.m]: Done.

**Pull request merged**